parity-crypto/Cargo.toml

```diff
@@ -18,7 +18,7 @@ aes = "0.6.0"
 aes-ctr = "0.6.0"
 block-modes = "0.7.0"
 digest = "0.9.0"
-ethereum-types = { version = "0.10.0", optional = true, path = "../ethereum-types" }
+ethereum-types = { version = "0.11.0", optional = true, path = "../ethereum-types" }
```

Comment:
I guess we will also need to bump `parity-crypto`

Reply:
yes done, thanks